adapter_ninafw.go

```diff
@@ -84,3 +83,48 @@ func resetNINAInverted() {
 	machine.NINA_RESETN.High()
 	time.Sleep(1000 * time.Millisecond)
 }
+
+type hciUART struct {
+	uart *machine.UART
+}
```

Comment:
Why is this a separate implementation from the one in adapter_hci_uart.go?

Reply:
The NinaFW is like a special case of the HCI UART interface. That is the reason, to make it a little easier to use more standardized ones such as the nrf dongles.